.github/workflows/build.yml

```diff
@@ -0,0 +1,37 @@
+name: Build project with Maven
+on:
+  pull_request: 
+  push: 
+  schedule:
+  - cron: '2 2 * * 1-5' # run nightly master builds on weekdays
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
should we pin to an exact version of Ubuntu to avoid accidental surprises when it gets upgraded?

Reply:
Good suggestion, I've just pushed the change :-)